presto-cache/src/test/java/com/facebook/presto/cache/alluxio/TestAlluxioCachingFileSystem.java

```diff
@@ -234,6 +315,21 @@ public void testInitialization()
         assertEquals(metricsDomain, conf.get("sink.jmx.domain", "bad result"));
     }
 
+    // TODO: update unit tests after CacheManager.reset() is available to avoid using reflection to modify singleton
+    private void resetCacheManager()
+            throws Exception
+    {
+        Field field = CacheManager.Factory.class.getDeclaredField("CACHE_MANAGER");
+        field.setAccessible(true);
```

Comment:
I understand that this would be made accessible in later versions?

Reply:
Correct, i already submitted https://github.com/Alluxio/alluxio/pull/12169 to expose this as an API